### ISSUE TITLE
Handle code failure in user submissions.

### ISSUE THREAD

**Issue by rwongone** [rwongone/testifi#78]

**Comment by matthewdu** [comment]
What type of failure exactly? If the user's submission results in a syntax error/compilation error, nothing catastrophic will happen. The execution model will just say the execution did not pass

**Comment by rwongone** [comment]
Yes, but the user gets no feedback with respect to what went wrong: all we see is that the output of the Submission is `nil`, we may want something more useful.
